CiarnuroRT/webapp/style.css

```diff
@@ -385,27 +385,100 @@ div.timer{
     text-align:center;
     overflow:show;
     white-space:nowrap;
-    font-size:1.35em;
+    font-size:1.5em;
 }
 div.mainTimerContainer > div.timer{
-    font-size:3em;
+    font-size:3.2rem;
 }
 div.smallTimerContainer > div.timerName{
-    font-size:1.05em;
+    font-size:1.05rem;
     text-align:center;
 }
 div.banner{
     text-align:center;
-    font-size:1.25em;
+    font-size:1.4em;
     margin-bottom:1em;
     font-weight:bold;
 }
 /*---"RESPONSIVE" WELCOME SCREEN---*/
-@media all and (max-width: 50em){
+@media all and (max-width: 50rem){
     #welcome > div.full{
         display:flex;
     }
     #welcome > div.full > div.content{
         margin:auto;
     }
 } 
+
+/*---IMAGE RESOLUTION---*/
+@media all and (max-width: 900px){
```

Comment:
This needs to be done in JS using the code you already added, checking the width in px doesn't work reliably in CSS.

Reply:
i removed this part and i made a function in javascript